contrib/gce/configure.sh

```diff
@@ -19,6 +19,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+PYTHON="python"
```

Comment:
you don't need this line :) 

looks like you picked the snippet below from:
https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L2895-L2909

So that's well battle-tested!

Reply:
yes, I picked from K8s repo, and we hit the issue on Feb.